### PR TITLE
wedding invitation lading page

### PR DESCRIPTION
**Pull request by kngcl**
The landing page has five sections headers, home, wedding, RSVP, jane-john and footer sections. The landing page actually serves as an invitation to people.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+this project is all about online invitation.
+
+## App
+
+![Home](assets/images/home.png)
+![Wedding](assets/images/wedding.png)
+![jane-john](assets/images/jane-john.png)
+![rsvp](assets/images/rsvp.png)
+
```
Comment (nganifaith) [inline Readme.md:10]: missing a brief project description.

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- To get a local copy up and running follow these simple example steps.
+- Clone this repository with `git@github.com:kngcl/wedding.git` using your terminal.
+- Change to the project directory by entering: cd food in the terminal.
+
+## steps
+
+- $ git clone `git@github.com:kngcl/wedding.git`
+- $ `cd wedding`
+- $ `git checkout feature`
```
Comment (nganifaith) [inline Readme.md]: which feature branch?

- [x] indicate feature branch that contains all your work

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Start App
+
+- run `npm install`
```
Comment (nganifaith) [inline Readme.md]: project doesn't require npm to run.

- [x] explain how to run project

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- run `npm start` in your CLI
+
+## Test
+
```
Comment (nganifaith) [inline Readme.md]: You are not working on any test. section isn't required.

- [x] remove test section

File: Readme.md (hunk 3 above)
Comment (nganifaith) [inline Readme.md]: Explain how to open project on browser

File: Readme.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- CSS
+
+## Clone project
+
+- To get a local copy up and running follow these simple example steps.
+- Clone this repository with `git@github.com:kngcl/wedding.git` using your terminal.
+- Change to the project directory by entering: cd food in the terminal.
+
+## steps
+
```
Comment (nganifaith) [inline Readme.md:31]: prerequisite section should have include:


- [x]  knowledge of HTML and CSS and 
- [x]  browser installed in order to run project

File: index.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="assets/styles/style.css" rel="stylesheet">
+    </head>
+    <body>
+          <!--Nav-bar-->
+        <section class="nav-bar">
+            <nav class="nav-heading">
+                <p><a href="#home">Home</a></p>
+                <p><a href="#jane-john">Jane and John</a></p>
```
Comment (nganifaith) [inline index.html]: Expected nav bar to be in a header tag

**Review by nganifaith — CHANGES_REQUESTED** [review]

**Comment by nganifaith** [comment]
Implement changes and submit for another review

File: index.html (hunk 6 above)
Comment (kngcl) [reply index.html]: Done

File: Readme.md (hunk 3 above)
Comment (kngcl) [reply Readme.md]: Done

File: Readme.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+## Clone project
+
+- To get a local copy up and running follow these simple example steps.
+- Clone this repository with `git@github.com:kngcl/wedding.git` using your terminal.
+- Change to the project directory by entering: cd wedding in the terminal.
+
+## steps
+
+- $ git clone `https://github.com/kngcl/wedding/tree/feature/header`
```
Comment (nganifaith) [inline Readme.md]: - [x] link to the repository

File: Readme.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+![Home](assets/images/home.png)
+![about-Wedding](assets/images/wedding.png)
+
+### Built With
+
+- HTML
+- CSS
+  
+### Prerequisites
+
```
Comment (nganifaith) [inline Readme.md:17]: what are the prerequisites to run or have the project install on your machine

File: Readme.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# wedding project
+
+this project is all about online invitation.
```
Comment (nganifaith) [inline Readme.md:3]: Give a description that can make someone want to visit the site or have it on their local machine

**Review by nganifaith — CHANGES_REQUESTED** [review]
@kngcl Great progress working on the review. You are on step away from having it approved.
Implement the changes and send me a message on slack

**Review by nganifaith — APPROVED** [review]
Congratulations 🍻🥁🥁🎉🎉🎉🎊🎊🎊🎊🎊🎉🎉🎊🎉🎊🎉🎉🎊🎉🎉🎉@kngcl. You can go ahead and merge your work. 

**Optional**
  

- [ ] footer section doesn't look like the original page

**Pull request merged**